test/viewmodels/SdesFileViewModelsSpec.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels
+
+import base.SpecBase
+import models.DDStatementType.{DutyDeferment, Excise, ExciseDeferment, Supplementary, UnknownStatementType}
+import models.FileFormat.Pdf
+import models.{DDStatementType, DutyDefermentStatement, DutyDefermentStatementFile, DutyDefermentStatementFileMetadata}
+import play.api.Application
+import play.api.i18n.Messages
+import viewmodels.SdesFileViewModels.DutyDefermentStatementFileViewModel
+
+class SdesFileViewModelsSpec extends SpecBase {
+
+  "DutyDefermentStatementFileViewModel.downloadLinkAriaLabel" should {
+
+    "return correct label for different defermentStatementType" in new Setup {
+
+      val exciseViewModel = new DutyDefermentStatementFileViewModel(excise)
```

Comment:
I think it's better to spilt the teams up for each label type so that if there is an issue it's easier to work out with ones are at fault. If you don't want to do this we could group the vals together at the top as per normal.

Reply:
It is also correct to write  all the different types in a single test (as it makes a logical unit) as when any error occurs, it could still be found by the line number of the test, also it saves memory cause it uses only one instance of Setup. 

But I have split the tests and moved the vals to Setup